### ISSUE TITLE
installation from binary repos

### ISSUE THREAD

**Issue by helgasoft** [r-wasm/webr#435]
Appreciation and gratitude for your work, WebR is a gamechanger ! 🏆
I have two questions related to installation.

Wondering **how often** those 20K [binary repo](https://repo.r-wasm.org/) packages are **re-compiled** to WebAssembly. Is there some trigger by package, or a general update schedule for all packages (monthly, etc.), or is it a non-scheduled batch job ?

There is a [footnote](https://docs.r-wasm.org/webr/latest/packages.html#fn1) in the docs about an alternative repo called [r-universe](https://r-universe.dev/).  Looks good indeed and they recompile really fast the latest package version when published. But trying to install from them I got and [reported an error](https://discuss.ropensci.org/t/webr-package-install/3916). Here is the code:
```
var webR;
import('https://webr.r-wasm.org/latest/webr.mjs').then(async ({ WebR }) => {
  webR = new WebR();
  await webR.init();
  // ok, but gets older version from repo.r-wasm.org :
  // await webR.installPackages(["echarty"])  

  // trying to get the latest version
  await webR.installPackages(["echarty"], {repos:'https://helgasoft.r-universe.dev'});  
ERROR 404:
https://helgasoft.r-universe.dev/bin/emscripten/contrib/4.4/PACKAGES.rds
Unsupported PACKAGES format: rds
```
To summarize the issue - _repo.r-wasm.org_ works fine, but packages are not updated often, _r-universe.dev_ has quick updates, but generates some installation error.

**Comment by georgestagg** [comment]
> Wondering how often those 20K [binary repo](https://repo.r-wasm.org/) packages are re-compiled to WebAssembly.

It's currently a non-scheduled batch job. Updates follow CRAN versions of packages but on a much slower release frequency. I can also update individual packages on request.

---

AFAICT it doesn't look like r-universe generates a `PACKAGES.rds` file, leading to the 404 error. However, R has fallback options, `PACKAGES.gz` and `PACKAGES`, which do exist. So, despite the error message, R should have continued on and loaded the package list from one of the fallback options.

For me, at least in a web browser, when I run

```
await webR.installPackages(["echarty"], {repos:'https://helgasoft.r-universe.dev'});
await webR.installPackages(["dplyr", "htmlwidgets", "data.tree"]);
```

The error is generated, but R continues to download using the fallback option and once complete I can successfully run

```
library(echarty)
```

I will try to think if there is some better way to raise and report the network error in this context.

**Comment by helgasoft** [comment]
> despite the error message, R should have continued on and loaded the package list from one of the fallback options.

Yes, correct.  &nbsp; &nbsp; Just FYI some performance data, _o3_ are (dplyr, htmlwidgets, data.tree)

loading | elapsed ms
--- | --- 
echarty only from r-wasm (o3 auto-loaded as dependencies) | 7,354
echarty from r-universe, o3 from r-wasm (your example) | 7,976
echarty and o3 (one-by-one), all from r-universe | 12,784

So loading from _r-wasm_ is almost twice as fast as _r-universe_ (at least from my PC and location).
BTW, thanks for updating _echarty_ to the latest version 🤙
